File: Desktop/DSA/T93/18.py
#swapalternatives 

def main(array:list[int])->list[int]:
    
    n=len(array)

    for i in range(0,n-1,2):
        
        array[i],array[i+1]=array[i+1],array[i]
        
    
    return array
arr = [1, 2, 3, 4, 5, 6]
print(main(arr))

File: Desktop/DSA/T93/18.js

//next smaller element

const  main=(array) => {

    let n = array.length
    let dp=Array(n).fill(-1)

    for (let i=0 ; i<n ; i++){

        for(let j=i+1;j<n;j++){

            if (array[j]<array[i]){

                dp[i]=array[j]
                break
            }
        }
    }

    return dp
}

const arr = [4, 5, 2, 10, 8];
console.log(main(arr));